app/sass/login-form.scss

```diff
@@ -0,0 +1,72 @@
+$base-color: #b73535;
+@mixin login-form-font($size) {
+  font: {
+    family: Georgia, Cambria, "Times New Roman", Times, serif;
+    style: italic;
+    size: $size;
+  };
+}
+.login-form {
+  margin-top: 100px;
+  display: flex;
+  justify-content: center;
+  input {
```

Comment:
Use of element as selectors is discouraged anywhere except in base layer. This could be moved to `base/_forms.scss` for example.